stress-example/nlp_search/pods/annoy_indexer.yml

```diff
@@ -0,0 +1,49 @@
+!CompoundIndexer
+components:
+  - !AnnoyIndexer
+    with:
+      #index_filename: 'vec.gz' # Should be overwriten by ref_indexer index filename
```

Comment:
Then should this be left here?

Reply:
good for clarification

Reply:
I thought it was good to left as to clarify why I'm not using that name